python/ray/tune/trainable.py

```diff
@@ -489,9 +488,15 @@ def _save(self, checkpoint_dir):
     def _restore(self, checkpoint):
         """Subclasses should override this to implement restore().
 
+         The directory structure is expected to be the same as that left by
```

Comment:
Can you provide some examples for save/restore? I find it hard to understand what the semantics are.

Reply:
OK that's a good point..

Reply:
still todo.